src/WebJobs.Extensions.DurableTask/Listener/OutOfProcOrchestrationShim.cs

```diff
@@ -124,9 +126,20 @@ private async Task ProcessAsyncActions(AsyncAction[][] actions)
                             tasks.Add(this.context.CallSubOrchestratorWithRetryAsync(action.FunctionName, action.RetryOptions, action.InstanceId, action.Input));
                             break;
                         case AsyncActionType.CallEntity:
-                            var entityId = EntityId.GetEntityIdFromSchedulerId(action.InstanceId);
-                            tasks.Add(this.context.CallEntityAsync(entityId, action.EntityOperation, action.Input));
-                            break;
+                            {
+                                var entityId = EntityId.GetEntityIdFromSchedulerId(action.InstanceId);
+                                tasks.Add(this.context.CallEntityAsync(entityId, action.EntityOperation, action.Input));
+                                break;
+                            }
+
+                        case AsyncActionType.SignalEntity:
+                            {
+                                // We do not add a task because this is 'fire and forget'
+                                var entityId = EntityId.GetEntityIdFromSchedulerId(action.InstanceId);
+                                this.context.SignalEntity(entityId, action.EntityOperation, action.Input);
```

Comment:
why not  support the more general version of signal  (scheduled signals) since you are at it? It's just one more argument being passed through. You can perhaps just reuse the FireAt field already there for the time.

Reply:
Ah, I didn't know about this variant. I'll take a stab at it

Reply:
Good catch Sebastian. I forgot all about that.

Reply:
I added the scheduled signal as a separate action code and tested via JavaScript that it can be reached